### ISSUE TITLE
Help! All of the default Nyquist plugins disappeared from Tenacity.

### ISSUE THREAD

**Issue by n0toose** [tenacityteam/tenacity-legacy#464]
### Guidelines

- [X] I have read the guidelines.

### Version/Commit hash

0.1.0, 9fd60e6

### Describe the bug.

The Limiter effect, which is present on Audacity and in our [source tree](https://github.com/tenacityteam/tenacity/blob/d28166652c33d9f81ac63ef3767852985049462b/plug-ins/limiter.ny), is not enabled in Tenacity.

### Expected behavior

The effect should appear normally and be usable.

### OS

All

### This issue is not a duplicate

- [X] I have confirmed this issue isn't a duplicate.

**Comment by n0toose** [comment]
This bug was reported to me by a social media user.

**Comment by Be-ing** [comment]
I think it's just missing from the list in nyquist/CMakeLists.txt.

**Comment by n0toose** [comment]
> I think it's just missing from the list in nyquist/CMakeLists.txt.

Checking...

**Comment by n0toose** [comment]
Seems to be present here:

https://github.com/tenacityteam/tenacity/blob/4408d96ff505d9d75de997ed5ca483a5ea393da5/plug-ins/CMakeLists.txt#L24

**Comment by n0toose** [comment]
f46015d63c09fe661402201ea6154ef9e179ebf7 does not seem to have broken the plug-ins.

They did not disappear, they are just disabled by default for some reason.

**Comment by n0toose** [comment]
4b8d4d30f8eb4a0d79210ecd76b46d84d6950256 (the commit before f46015d) and the HEAD seems to be fine too, even though I had to delete `~/.local/share/tenacity` and `~/.config/tenacity`.

After 50 minutes of painstaking agony and trying to figure out how this could've possibly happened with @SFR-git and @Semisol, we only have a big "I don't know how this happened" - This does not happen on the master branch, there's a decent chance that this just randomly broke at some point and produced broken configuration files for our early adopters. However, we figured out a way to fix this!

Here's the fix:

Tools -> Add / Remove Plug-ins -> Click on the Disabled circle thingie -> Enable every plugin that shows up.
